src/node/types.h

```diff
@@ -39,6 +41,13 @@ struct BlockCreateOptions {
      * larger CompactSize encoded lengths.
      */
     size_t block_reserved_weight{8000};
+
+    /*
+     * This is a sane minimum value for block_reserved_weight.
```

Comment:
Maybe add:

> This accounts for the block header, var_int encoding of the transaction count and a minimally viable coinbase transaction. It adds an additional safety margin, because even with a thorough understanding of block serialization, it's easy to make a costly mistake when trying to squeeze every last byte.